demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -360,6 +360,19 @@ def get_display_name(file_path) -> str:
             name = file_data.get('brandName', None)
         elif 'id' in file_data:
             name = file_data.get('id', None)
+        elif 'trigger_id' in file_data:
+            name = file_data.get('trigger_id')
+
+        elif 'dashboards_data' in file_data and file_data.get('dashboards_data') \
+                and isinstance(file_data['dashboards_data'], list):
+            dashboard_data = file_data.get('dashboards_data', [])[0]
```

Comment:
By default it will fail on `list index out of range`.

Maybe:
```suggestion
            dashboard_data = file_data.get('dashboards_data', [{}])[0]
```